staging/nvidia/charts/nvidia-driver/templates/daemonset.yaml

```diff
@@ -43,10 +44,28 @@ spec:
         - name: driver-path
           mountPropagation: Bidirectional
           mountPath: /run/nvidia
+        - name: modules
+          mountPath: /lib/modules
+        - name: vault-repo
+          mountPath: /etc/yum.repos.d/CentOS-Vault.repo
+          subPath: CentOS-Vault.repo
+        - name: wrapper-sh
+          mountPath: /usr/local/bin/wrapper.sh
+          subPath: wrapper.sh
       volumes:
       - name: update-driver
         hostPath:
           path: /etc/kernel/postinst.d
       - name: driver-path
         hostPath:
           path: /run/nvidia
+      - name: modules
+        hostPath:
+          path: /lib/modules
```

Comment:
is there a way to mount this as RO? I do see `rm -rf /lib/modules/${KERNEL_VERSION}` in the driver init script, but I didn't check deeper. Allowing the container to delete files on the host under `/lib/modules` sounds very dangerous to me.

Reply:
More dangerous than installing a kernel module?

Reply:
I will check the init script more carefully today. I just don’t want the files in modules dir to be accidentally deleted as I do see some rm in the script.

Kernel module is trusted in this case. But the script might assume modules dir in the container is decoupled from the host